core-bundle/contao/languages/en/default.xlf

```diff
@@ -2472,10 +2484,16 @@
         <source>Collapse all nodes</source>
       </trans-unit>
       <trans-unit id="DCA.toggleFilter.0">
-        <source>Filter</source>
+        <source>Show filters</source>
       </trans-unit>
       <trans-unit id="DCA.toggleFilter.1">
-        <source>Show the filters</source>
+        <source>Show the DCA filter panel</source>
+      </trans-unit>
+      <trans-unit id="DCA.toggleFilter.2">
+        <source>Hide filters</source>
+      </trans-unit>
+      <trans-unit id="DCA.toggleFilter.3">
+        <source>Hide the DCA filter panel</source>
```

Comment:
```suggestion
        <source>Show the filter panel</source>
      </trans-unit>
      <trans-unit id="DCA.toggleFilter.2">
        <source>Hide filters</source>
      </trans-unit>
      <trans-unit id="DCA.toggleFilter.3">
        <source>Hide the filter panel</source>
```

Normal users do not understand the concept of `DCA`, they do however know what a "filter panel" is

Reply:
I disagree on the title, because the button can still be reached using the keyboard, and then it should say what it does. But imporoved in 2112baf5214450b95bff21a7e8257ff939aa47f1